src/test/java/be/syntra/devshop/DevshopFront/testutils/ProductUtils.java

```diff
@@ -25,6 +25,20 @@ public static ProductDto getDummyProductDto() {
                 .build();
     }
 
+    public static ProductDto getDummyRatedProductDto() {
```

Comment:
this is exactly the same as getDummyNonArvichedProduct. So this new one can be removed

Reply:
ProductDto <-> Product
changed above name to getDummyNonArchivedProductDto()

Reply:
What we're trying to achieve here, is to avoid code-duplication. If there are two different methods, doing the same, that's confusing. Yes, I mentioned the wrong referencemethod, but if you'd added the same fields, enlarging getDummyProductDto(), it would be still ok to remove the new method.

Reply:
now i get it :-)